dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -515,6 +515,22 @@ fn print_task_details(bgtask: &BackgroundTask, details: &serde_json::Value) {
                 );
             }
         };
+    } else if name == "phantom_disks" {
+        #[derive(Deserialize)]
+        struct TaskSuccess {
+            /// how many phantom disks were found
+            ok: usize,
```

Comment:
A more accurate name could be helpful, maybe `n_disks` or `disk_count`?

Reply:
agreed, this was me blindly copying from the background task output. done in e64ec0850